integrations/nielsen-dcr/lib/index.js

```diff
@@ -409,93 +428,47 @@ NielsenDCR.prototype.videoPlaybackSeekCompleted = function(track) {
-  var assetId = contentAssetId || adAssetId;
+  // you should _only_ be sending the asset_id of whatever you are resuming in: content or ad
   var type = contentAssetId ? 'content' : 'ad';
+  var assetId = contentAssetId || adAssetId;
 
   if (this.currentAssetId && this.currentAssetId !== assetId) {
     if (type === 'ad') {
       this._client.ggPM('loadMetadata', this.getAdMetadata(track));
     } else if (type === 'content') {
-      this._client.ggPM(
-        'loadMetadata',
-        this.getContentMetadata(track, 'video')
-      );
+      this._client.ggPM('loadMetadata', this.getContentMetadata(track));
     }
   }
 
   this.heartbeat(assetId, position, {
     type: type,
     livestream: livestream,
-    timestamp: track.timestamp()
+    timestamp: timestamp
   });
 };
 
-/**
- * Video Playback Paused
- *
- * @api public
- */
-
-NielsenDCR.prototype.videoPlaybackPaused = function(track) {
-  clearInterval(this.heartbeatId);
-  // NOTE: docs say to just pause the pings but Nielsen Reps say to call `stop`
-  this._client.ggPM('stop', track.proxy('properties.position'));
-};
-
-/**
- * Video Playback Resumed
- *
- * @api public
- */
-
-NielsenDCR.prototype.videoPlaybackResumed = function(track) {
-  clearInterval(this.heartbeatId);
-
-  var contentAssetId = this.options.contentAssetIdPropertyName
-    ? track.proxy('properties.' + this.options.contentAssetIdPropertyName)
-    : track.proxy('properties.content_asset_id');
-  var adAssetId = this.options.adAssetIdPropertyName
-    ? track.proxy('properties.' + this.options.adAssetIdPropertyName)
-    : track.proxy('properties.ad_asset_id');
-  var position = track.proxy('properties.position');
-  // if properly implemented, the point in which the playback is resumed
-  // you should _only_ be sending the asset_id of whatever you are resuming in: content or ad
-  var type = contentAssetId ? 'content' : 'ad';
-  var assetId = contentAssetId || adAssetId;
-
-  if (this.currentAssetId && this.currentAssetId !== assetId) {
-    if (type === 'ad') {
-      this._client.ggPM('loadMetadata', this.getAdMetadata(track));
-    } else if (type === 'content') {
-      this._client.ggPM(
-        'loadMetadata',
-        this.getContentMetadata(track, 'video')
-      );
-    }
-  }
-
-  this.heartbeat(assetId, position, { type: type });
-};
-
 /**
  * Video Playback Completed
+ * Video Playback Interrupted
  *
  * @api public
  */
 
-NielsenDCR.prototype.videoPlaybackCompleted = function(track) {
+NielsenDCR.prototype.videoPlaybackCompleted = NielsenDCR.prototype.videoPlaybackInterrupted = function(
+  track
+) {
   clearInterval(this.heartbeatId);
 
-  var position = track.proxy('properties.position');
+  var timestamp = track.timestamp();
   var livestream = track.proxy('properties.livestream');
-
-  // for livestream just send the current utc timestamp
-  if (livestream) position = +Date.now(track.timestamp());
+  var position = livestream
+    ? +Date.now(timestamp)
```

Comment:
I don't think `Data.now` accepts an argument. This will always just return the current ms since the epoch regardless of the value of `timestamp`